Sources/Valet/SecureEnclaveValet.swift

```diff
@@ -125,6 +135,9 @@ public final class SecureEnclaveValet: NSObject {
     /// - returns: The data currently stored in the keychain for the provided key. Returns `.itemNotFound` if no object exists in the keychain for the specified key, or if the keychain is inaccessible. Returns `.userCancelled` if the user cancels the user-presence prompt.
     public func object(forKey key: String, withPrompt userPrompt: String) -> SecureEnclave.Result<Data> {
         return execute(in: lock) {
+            guard let keychainQuery = keychainQuery else {
+                return .itemNotFound
```

Comment:
Should we add a new result case for `.keychainNotAccessible`?  (applies throughout diff)

Reply:
Per the comment above:
> Returns `.itemNotFound` if no object exists in the keychain for the specified key, or if the keychain is inaccessible.

So we're doing the right thing per the comment. Adding a new case here would be a breaking change. The Valet 4.0 code will `throw` the right error (`couldNotAccessKeychain `) in this case assuming #198 lands